.mergify.yml

```diff
@@ -319,6 +306,14 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: backport patches to 8.19 branch
+    conditions:
+      - merged
+      - label=backport-8.19
+    actions:
+      backport:
+        branches:
```

Comment:
Use default values:
- https://github.com/elastic/fleet-server/blob/cc0972901ad62ab25b148f8efd619d1373b57aa6/.mergify.yml#L12-L19

In a follow-up we can reduce the explicit duplicated entries per backport rule